contracts/pcv/PCVSwapperUniswap.sol

```diff
@@ -247,11 +247,13 @@ contract PCVSwapperUniswap is IPCVSwapper, OracleRef, Timed {
       uint256 n;
       bool direction;
       if (decimalsTokenSpent >= decimalsTokenReceived) {
+        require(decimalsTokenSpent - decimalsTokenReceived <= 77, "PCVSwapperUniswap: Decimal overflow.");
```

Comment:
Probably not required as the first deployments of this contract will use safe ERC20s and we'll have version 0.8 soon

Reply:
Do you mind if we leave it ?

Reply:
Sure we can leave it